.github/workflows/e2e_trigger_via_label.yaml

```diff
@@ -0,0 +1,21 @@
+name: trigger e2e on label
+
+on:
+  pull_request:
+    types:
+    - labeled
+
+jobs:
+  trigger-e2e-tests:
+    if: contains(github.event.*.labels.*.name, 'ci/run-e2e')
+
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.K8S_TEAM_BOT_GH_PAT }}
+      WORKFLOW: .github/workflows/e2e.yaml
```

Comment:
I think that if the intention is to run E2E tests against the KIC image built from the ref source, we should instead call `targeted_e2e.yaml` as the regular one always runs against the nightly image.

Reply:
That actually makes sense. I've made it so that it runs e2e targeted then. 👍